changelog/fragments/1744831791-Add-version-metadata-to-version-command-output.yaml

```diff
@@ -0,0 +1,34 @@
+# Kind can be one of:
```

Comment:
Are we adding CHANGELOGS for FIPS work? See discussion in https://github.com/elastic/elastic-agent/pull/7862#discussion_r2045574930

Reply:
Normally no, but this adds the commit hash and buildtime to the version command as well